code/modules/zombie/organs.dm

```diff
@@ -80,12 +78,12 @@
 	//Fully heal the zombie's damage the first time they rise
 	C.setToxLoss(0, 0)
 	C.setOxyLoss(0, 0)
+	C.adjustOrganLoss(ORGAN_SLOT_BRAIN, -200)
```

Comment:
Maybe this should adjust by -max health of the brain, rather than -200? In case we get brains with different max healths.

Reply:
Good catch, zombies do also heal brain damage over time now so this would probably correct itself, but that would be much more proper.

Reply:
fixed and also pushed some changes I forgot to push before sleeping